vochain/indexer/indexer.go

```diff
@@ -705,3 +715,78 @@ func (idx *Indexer) GetTokenFeesByType(txType string, offset, maxItems int32) ([
 	}
 	return tt, nil
 }
+
+// GetTokenTransfersByToAccount returns all the token transfers made to a given account
+// from the database, ordered by timestamp and paginated by maxItems and offset
+func (idx *Indexer) GetTokenTransfersByToAccount(to []byte, offset, maxItems int32) ([]*indexertypes.TokenTransferMeta, error) {
```

Comment:
what about dropping the "By", could it be GetTokenTransfersToAccount ?
that would make also GetTokenTransfersFromAccount
and the very explicit GetTokenTransfersFromOrToAccount

Reply:
In the indexer methods the "By" is currently used in: `GetProcessIDsByFinalResults, GetTokenFeesByFromAccount, UpdateProcessResultByID`. In order to keep the pattern I think we can also use it here.

Reply:
perfect, SGTM